CHANGELOG.md

```diff
@@ -1,5 +1,99 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 4.12.0
+###### _Nov 14 2020_
+
+Big thanks to the 10 contributors who made this release possible.
+
+This release is intended to help and prepare the migration to Material-UI v5 ⏫:
+
+- 📚 Add deprecations to anticipate v5. A deprecation is added every time a breaking change in v5 allows it. If no warnings are reported in the console, you are set. Each warning comes with a simple message that explains how to handle the deprecation.
```

Comment:
- 📚 Start to add deprecations in anticipation of v5. We plan to add a deprecation for any breaking change in v5 that allows it. Each warning comes with a simple message that explains how to handle the deprecation. If no warnings are reported in the console, you are set for this first batch.

Reply:
And get rid of the "simple message". They're more often than not not simple.